src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
     assertSingleArgument(params).map {
       case f: WdlFile =>
-        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.toInt)
+        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.trim.toInt)
```

Comment:
You have to call `close` yourself. It's registering the function which gets called when you say `foo.close`.

I've typically seen people do this (which irks me as it removes the convenience of `fromFile`):

    val foo = Source.fromFile(.........)
    // do something
    foo.close

Although I saw @scottfrazer do something clever the other day involving a finally all on the same line. I've seen the advice given that `fromFile` is really intended to be a lightweight convenience method when noodling around and to stick w/ the java file readers as long as ARM isn't involved but that also irks me.